source/agora/node/FullNode.d

```diff
@@ -229,6 +254,7 @@ public class FullNode : API
         this.pool = null;
         this.utxo_set = null;
         this.enroll_man = null;
+        this.http_listener.stopListening();
```

Comment:
This doesn't really belong here. We do not use any networking in unittest builds. This could be moved here:

https://github.com/bpfkorea/agora/blob/4fc127860126d78610676c92d49fdad55576aa66/source/agora/node/main.d#L84

And then the HTTPListener could be returned from `runNode` or set through a parameter, and `main()` can call `stopListening` on it after it calls `node.shutdown()`.